sources/packages/api/src/cra-integration/cra-integration.models.ts

```diff
@@ -76,6 +76,68 @@ export enum MatchStatusCodes {
   unsuccessfulMatch = "50",
 }
 
+/**
+ * SIN match status codes (SIN-TTN-STATUS-CODE) presents on
+ * CRA Response Record (Trans Sub Code - 0022).
+ */
+export enum SINMatchStatusCodes {
+  SINTTNStatusNotSet = "00",
+  SINTTNMatched = "01",
+  SINTTNUnderANM = "02",
+  SINTTNNotFound = "50",
+  SINTTNPendingANM = "51",
+  SINTTNMissing = "90",
+  SINTTNInvalidFormat = "91",
+}
+
+/**
+ * Surname match status codes (SURNAME-STATUS-CODE) presents on
+ * CRA Response Record (Trans Sub Code - 0022).
+ */
+export enum SurnameStatusCode {
+  SurnameStatusNotSet = "00",
+  SurnameMatchedCRA = "01",
+  PreviousSurnameMatched = "02",
+  CiSurnameMatched = "03",
+  SurnameNotMatched = "50",
+  SurnameMissing = "90",
+}
+
+/**
+ * GivenName match status codes (GIVEN-NAME-STATUS-CODE) presents on
+ * CRA Response Record (Trans Sub Code - 0022).
+ */
+export enum GivenNameStatusCode {
+  GivenNameStatusNotSet = "00",
+  GivenNameMatchedCRA = "01",
+  PreviousGivenNameMatched = "02",
+  CiGivenNameMatched = "03",
+  GivenNameNotMatched = "50",
+  GivenNameMissing = "90",
+}
+
+/**
+ * BirthDate match status codes (BIRTH-DATE-STATUS-CODE) presents on
+ * CRA Response Record (Trans Sub Code - 0022).
+ */
+export enum BirthDateStatusCode {
```

Comment:
When we have s long list, can we make use of a table instead of enum.? Probably all the 3 enums in a single table?

Reply:
Discussed in call